Bottom: c3825de3e4fb1ef6cf4c695042c192637f80aa48
Top:    7de610435d079754bc65f39c3f216185b90b6e5b
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-12-17 02:26:07 +0000

TEMPORARY: Comment out some DrawMark calls which might be out of context

Should fix this properly with notificatons to the GUI


---

diff --git a/src/action.c b/src/action.c
index e3576bf..657b337 100644
--- a/src/action.c
+++ b/src/action.c
@@ -4020,7 +4020,7 @@ ActionMarkCrosshair (int argc, char **argv, int x, int y)
     {
       if (Marked.status)
 	{
-	  DrawMark (true);
+//	  DrawMark (true);
 	  Marked.status = false;
 	}
       else
@@ -4028,16 +4028,16 @@ ActionMarkCrosshair (int argc, char **argv, int x, int y)
 	  Marked.status = true;
 	  Marked.X = Crosshair.X;
 	  Marked.Y = Crosshair.Y;
-	  DrawMark (false);
+//	  DrawMark (false);
 	}
     }
   else if (GetFunctionID (function) == F_Center)
     {
-      DrawMark (true);
+//      DrawMark (true);
       Marked.status = true;
       Marked.X = Crosshair.X;
       Marked.Y = Crosshair.Y;
-      DrawMark (false);
+//      DrawMark (false);
     }
   return 0;
 }
